var/spack/repos/builtin/packages/neuron/package.py

```diff
@@ -63,6 +63,7 @@ class Neuron(CMakePackage):
 
     variant("cmake",      default=True, description="Build NEURON using cmake")
     variant("binary",     default=True, description="Create special as a binary instead of shell script (until 8.0.0)")
+    conflicts("+binary", when='@8.0.1:')
```

Comment:
```suggestion
    conflicts("~binary", when='@8.0.1:')
```
Sorry, just realized that the default is `+binary`, shouldn't ban that